<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StampStripDialog</class>
 <widget class="QDialog" name="StampStripDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>422</width>
    <height>735</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>422</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>422</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Strip Stamp Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="frameStampLayoutLabel">
       <property name="text">
        <string>Strip Stamp Layout</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="frameStampLayoutComboBox">
       <property name="styleSheet">
        <string notr="true">QComboBox QAbstractItemView::item {margin:2px;}</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="FrameGroupBox">
     <property name="maximumSize">
      <size>
       <width>400</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox#FrameGroupBox{border: 1px solid rgb(180, 180, 180)}</string>
     </property>
     <property name="title">
      <string>Frame Property</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <property name="spacing">
       <number>10</number>
      </property>
      <property name="leftMargin">
       <number>11</number>
      </property>
      <property name="topMargin">
       <number>15</number>
      </property>
      <property name="rightMargin">
       <number>11</number>
      </property>
      <property name="bottomMargin">
       <number>11</number>
      </property>
      <item>
       <layout class="QFormLayout" name="formLayout">
        <property name="labelAlignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="formAlignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
        <property name="verticalSpacing">
         <number>10</number>
        </property>
        <property name="topMargin">
         <number>5</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="frameWidthLabel">
          <property name="text">
           <string>Width</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="LaserDoubleSpinBox" name="frameWidthSpinBox">
          <property name="minimumSize">
           <size>
            <width>70</width>
            <height>0</height>
           </size>
          </property>
          <property name="suffix">
           <string>mm</string>
          </property>
          <property name="decimals">
           <number>3</number>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="frameHeightLabel">
          <property name="text">
           <string>Height</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="LaserDoubleSpinBox" name="frameHeightSpinBox">
          <property name="minimumSize">
           <size>
            <width>70</width>
            <height>0</height>
           </size>
          </property>
          <property name="suffix">
           <string>mm</string>
          </property>
          <property name="decimals">
           <number>3</number>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="frameBorderLabel">
          <property name="text">
           <string>Border</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="LaserDoubleSpinBox" name="frameBorderSpinBox">
          <property name="suffix">
           <string>mm</string>
          </property>
          <property name="decimals">
           <number>3</number>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="frameCornerTypeLabel">
          <property name="text">
           <string>Corner Type</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QComboBox" name="frameCornerTypeComboBox"/>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="frameCornerSizeLabel">
          <property name="text">
           <string>Corner Size</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="LaserDoubleSpinBox" name="frameCornerSizeSpinBox">
          <property name="minimumSize">
           <size>
            <width>70</width>
            <height>0</height>
           </size>
          </property>
          <property name="suffix">
           <string>mm</string>
          </property>
          <property name="decimals">
           <number>3</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="innerFrameGroupBox">
        <property name="title">
         <string>Inner Frame</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <layout class="QFormLayout" name="innerFrameLayout">
           <property name="labelAlignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="innerFrameMarginLabel">
             <property name="text">
              <string>Inner Frame Margins</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="LaserDoubleSpinBox" name="InnerFrameMarginDoubleSpinBox">
             <property name="suffix">
              <string>mm</string>
             </property>
             <property name="decimals">
              <number>3</number>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="innerFrameBorderLabel">
             <property name="text">
              <string>Inner Frame Border</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="LaserDoubleSpinBox" name="InnerFrameBorderDoubleSpinBox">
             <property name="suffix">
              <string>mm</string>
             </property>
             <property name="decimals">
              <number>3</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="textPropertyGroupBox">
     <property name="minimumSize">
      <size>
       <width>400</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>400</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox#textPropertyGroupBox{border: 1px solid rgb(180, 180, 180)}</string>
     </property>
     <property name="title">
      <string>Text Property</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>15</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <property name="horizontalSpacing">
       <number>10</number>
      </property>
      <property name="verticalSpacing">
       <number>5</number>
      </property>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>5</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QToolButton" name="boldBtn">
          <property name="text">
           <string>Bold</string>
          </property>
          <property name="icon">
           <iconset resource="../res/icons.qrc">
            <normaloff>:/ui/icons/images/bold.png</normaloff>:/ui/icons/images/bold.png</iconset>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="itaticBtn">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../res/icons.qrc">
            <normaloff>:/ui/icons/images/straight.png</normaloff>:/ui/icons/images/straight.png</iconset>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="fillBtn">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../res/icons.qrc">
            <normaloff>:/ui/icons/images/fill.png</normaloff>:/ui/icons/images/fill.png</iconset>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QFontComboBox" name="fontComboBox">
          <property name="maximumSize">
           <size>
            <width>160</width>
            <height>25</height>
           </size>
          </property>
          <property name="editable">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QToolButton" name="addToolButton">
          <property name="toolTip">
           <string>add</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../res/icons.qrc">
            <normaloff>:/ui/icons/images/add_1.png</normaloff>:/ui/icons/images/add_1.png</iconset>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignRight">
         <widget class="QToolButton" name="delToolButton">
          <property name="toolTip">
           <string>delete</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../res/icons.qrc">
            <normaloff>:/ui/icons/images/delete_1.png</normaloff>:/ui/icons/images/delete_1.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="8" column="0">
       <widget class="QLineEdit" name="lineEdit">
        <property name="maximumSize">
         <size>
          <width>400</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="maxLength">
         <number>1000</number>
        </property>
       </widget>
      </item>
      <item row="11" column="0">
       <layout class="QFormLayout" name="formLayout_3">
        <property name="labelAlignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="formAlignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
        <property name="horizontalSpacing">
         <number>10</number>
        </property>
        <property name="verticalSpacing">
         <number>11</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <item row="1" column="0">
         <widget class="QLabel" name="HMarginLabel">
          <property name="text">
           <string>Horinzontal Margins</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="LaserDoubleSpinBox" name="hMarginSpinBox">
          <property name="suffix">
           <string>mm</string>
          </property>
          <property name="decimals">
           <number>3</number>
          </property>
          <property name="maximum">
           <double>98.989999999999995</double>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="VMarginlabel">
          <property name="text">
           <string>Vertical Margins</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="LaserDoubleSpinBox" name="vMarginSpinBox">
          <property name="suffix">
           <string>mm</string>
          </property>
          <property name="decimals">
           <number>3</number>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="hSpacelabel">
          <property name="text">
           <string>Horinzontal Spacing</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="LaserDoubleSpinBox" name="hSpaceSpinBox">
          <property name="suffix">
           <string>mm</string>
          </property>
          <property name="decimals">
           <number>3</number>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="vSpacelabellabel">
          <property name="text">
           <string>Vertical Spacing</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="LaserDoubleSpinBox" name="vSpaceSpinBox">
          <property name="suffix">
           <string>mm</string>
          </property>
          <property name="decimals">
           <number>3</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="9" column="0">
       <widget class="QTableView" name="tableView">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>142</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>142</height>
         </size>
        </property>
        <property name="autoScrollMargin">
         <number>16</number>
        </property>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderStretchLastSection">
         <bool>false</bool>
        </attribute>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LaserDoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>widget/LaserDoubleSpinBox.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../res/icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>StampStripDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>StampStripDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
